packages/components/src/ActionBar/ActionBar.component.js

```diff
@@ -223,24 +222,22 @@ export function ActionBarComponent(props) {
 	);
 }
 
-ActionBarComponent.propTypes = {
+ActionBar.propTypes = {
 	selected: PropTypes.number,
 	children: PropTypes.node,
 	className: PropTypes.string,
 	getComponent: PropTypes.func,
-	t: PropTypes.func,
 	components: PropTypes.object,
 };
-ActionBarComponent.defaultProps = {
+ActionBar.defaultProps = {
 	components: {},
 };
-ActionBarComponent.displayName = 'ActionBar';
-
-const TranslatedActionBar = withTranslation(I18N_DOMAIN_COMPONENTS)(ActionBarComponent);
-TranslatedActionBar.DISPLAY_MODES = DISPLAY_MODES;
-TranslatedActionBar.Count = Count;
-TranslatedActionBar.SwitchActions = SwitchActions;
-TranslatedActionBar.getActionsToRender = getActionsToRender;
-TranslatedActionBar.Content = Content;
-TranslatedActionBar.getContentClassName = getContentClassName;
-export default TranslatedActionBar;
+ActionBar.displayName = 'ActionBar';
+
+ActionBar.DISPLAY_MODES = DISPLAY_MODES;
+ActionBar.Count = Count;
+ActionBar.SwitchActions = SwitchActions;
+ActionBar.getActionsToRender = getActionsToRender;
+ActionBar.Content = Content;
+ActionBar.getContentClassName = getContentClassName;
+export default ActionBar;
```

Comment:
Removed the HOC, where the ActionBar pass the `t()` to only `Count`. Use directly i18next in `Count` via hook. That will simplify ActionBar  selections in tests